frontend/.eslintrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     ecmaFeatures: {
       jsx: true,
     },
-    ecmaVersion: 2018,
+    ecmaVersion: 2020,
```

Comment:
@positiveimpact I bumped this to support optional chaining. I think we should probably update our lint config in the future, but seems low priority.